### DISCUSSION TITLE
multiple prompts to generate images

### DISCUSSION THREAD

**Discussion by zmy-08** [PixArt-alpha/PixArt-alpha#110] [Q&A] [answered]
How do I use multiple prompts to generate images at once? Now you can only use one text at a time to generate images.

**Comment by lawrence-cj** [comment, marked as answer]
Just put your multiple prompts into a list, then send to your pipeline, like: https://huggingface.co/docs/diffusers/api/pipelines/pixart#diffusers.PixArtAlphaPipeline.__call__.prompt

For example:

```python
import torch
from diffusers import PixArtAlphaPipeline

# You can replace the checkpoint id with "PixArt-alpha/PixArt-XL-2-512x512" too.
pipe = PixArtAlphaPipeline.from_pretrained("PixArt-alpha/PixArt-XL-2-1024-MS", torch_dtype=torch.float16)

prompt = ["dog.", "cat"]
image = pipe(prompt).images[0]
```

**Reply by zmy-08** [reply]
Thank you very much!!!
